godot/src/ui/components/pointer_tooltip/pointer_tooltip.gd

```diff
@@ -13,13 +13,17 @@ func set_pointer_data(interacts_array: Array):
 		child.queue_free()
 	var i = 0
 	for interact in interacts_array:
+		if i >= angles.size():
```

Comment:
Does this discard the remaining interactions?

Reply:
Yes. It avoids a crash. In the Foundation Client is crashing. And Godot was crashing too due to not having enough angles.
There is no limit to how many tooltip interactions you can have. 5 it's more than ok.